newsletter-march-2023.rst

```diff
@@ -0,0 +1,127 @@
+  The separate landing page on readthedocs.org will also be moving.
+- 📚️ Documentation refactor underway:
+  A regular reader of our :doc:`user docs <readthedocs:index>` might notice that our navigation has been greatly changed and now contains only the `4 Diátaxis categories <https://diataxis.fr/>`__.
+- 🧹️ We are now able to delist projects from search engines more efficiently.
+  If you find an unofficial and unmaintained project,
+  :doc:`please see our policy on unofficial and unmaintained projects <readthedocs:unofficial-projects>`.
+- 🐞 Custom 404s with Sphinx DirHTML builder `are now supported <https://github.com/readthedocs/sphinx-notfound-page/issues/215>`__.
+- 🐞 :doc:`Pull Request <readthedocs:guides/pull-requests>` builds now link directly to the preview version of the documentation changes.
+- 🐞 A brief occurrence of redirect loops around February 13-14 `was resolved <https://github.com/readthedocs/readthedocs.org/issues/10017>`__.
+- 🔒️ Vulnerability fixed: Pull requests preview builds `GHSA-h4cf-8gv8-4chf <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-h4cf-8gv8-4chf>`__
+- 🔒️ Vulnerability fixed: Access to files from any project possible with crafted path traversal, see: `GHSA-5w8m-r7jm-mhp9 <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-5w8m-r7jm-mhp9>`__.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. _rclone: https://rclone.org/
+
+
+Upcoming features
+-----------------
+
+..
+  Notes:
+
+  Next newsletter:
+  Make a general announcement of our Roadmap
+
+  General:
+
+  When creating newsletter drafts, we keep the items here from the previous newsletter.
+  This is in order to ensure due follow-up on features that are announced publicly.
+  
+  Feature done? A great follow-up is to add what was previously an upcoming feature as a released feature in the former section.
+  
+  Feature not done?
+  Make sure that upcoming features are announced with a link to issues or PRs where the progress can be seen.
+  If this is done, then subsequent newsletters aren't compelled to share progress when it's uninteresting.
+  
+  If a feature was announced as upcoming but isn't yet released,
+  then try rephrasing the announcement as a general news update about the progress and where it can be followed.
+
+- 🪄️ The Django application that handles uncached requests on all our hosted documentation is called El Proxito.
+  And that application is getting a major refactor,
+  improving both speed and features.
+  Most of the refactoring work is already merged and within the coming weeks,
+  we will start to roll it out as a kind of A/B testing.
+- ⏳️ The first work from our Diátaxis documentation refactor is now live and the general work has moved into shorter iterations.
```

Comment:
Seems we're mentioning this twice? The above mention feels short, so maybe remove it, and just keep this longer explanation?

Reply:
Consolidated with the other item